saml/response.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	"github.com/jonboulle/clockwork"
 	saml2 "github.com/russellhaering/gosaml2"
+	"github.com/russellhaering/gosaml2/types"
```

Comment:
Can we use the `cap/saml` types instead? https://github.com/hashicorp/cap/blob/main/saml/models/core/response.go#L10-L14

Reply:
done